internal/pkg/model/naming.go

```diff
@@ -32,8 +32,8 @@ type Naming struct {
 	ConfigRow           utils.PathTemplate `json:"configRow" validate:"required"`
 	SharedCodeConfig    utils.PathTemplate `json:"sharedCodeConfig" validate:"required"`
 	SharedCodeConfigRow utils.PathTemplate `json:"sharedCodeConfigRow" validate:"required"`
-	Variables           utils.PathTemplate `json:"variables" validate:"required"`
-	VariablesValues     utils.PathTemplate `json:"variablesValues" validate:"required"`
+	VariablesConfig     utils.PathTemplate `json:"variablesConfig" validate:"required"`
```

Comment:
V tomto PR iba rename 2 fiedov, .... aby to bolo jednotne, vid `SharedCodeConfig` vyssie (podobny pripad ako variables).